forms-web-app/src/views/eligibility/planning-department.njk

```diff
@@ -2,11 +2,44 @@
 
 {% block content %}
 
-<div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
-    <span class="govuk-caption-l">Before you start</span>
-    <form action="">
-      {% from "govuk/components/input/macro.njk" import govukInput %}
+
+  {% if errors %}
+    <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1"
+         data-module="govuk-error-summary">
+      <h2 class="govuk-error-summary__title" id="error-summary-title">
+        There is a problem
+      </h2>
+      <div class="govuk-error-summary__body">
+        <ul class="govuk-list govuk-error-summary__list">
+          <li>
+            <a href="#department-error">Select the local planning department from the list</a>
+          </li>
+        </ul>
+      </div>
+    </div>
+  {% endif %}
```

Comment:
`errorSummary` takes care of this for you.

You should be able to replace with:

```
{% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
```
and
```
    {% if errorSummary %}
      {{ govukErrorSummary({
        titleText: "There is a problem",
        errorList: errorSummary
      }) }}
    {% endif %}
```